composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "minter/minter-php-bip-44",
```

Comment:
You can use your fork. Do not replace the name of the project in the composer.

Reply:
ok